style.css

```diff
@@ -4,6 +4,71 @@
   width: 100vw;
 }
 
+.hr-main {
+  background-color: #ec5242;
+  width: 6%;
+  position: relative;
+  top: 3rem;
+  height: 3px;
+  border: none;
```

Comment:
you have done good on this project , but all positioning issues you have are due to the use of these properties position: relative or absolute are used when we want to make an element overlap the element in dom flow , to fix this use only margin and padding to add spaces between elements and layout your components with flexbox and grid layouts 

![footer capstone](https://user-images.githubusercontent.com/59707859/226135293-a7a716a8-d994-41b4-b447-76fdc7df5e20.PNG)
![headline capstone](https://user-images.githubusercontent.com/59707859/226135297-9f0b506d-d073-43da-b23f-6fe9cd602d4f.PNG)
![speakers capstone](https://user-images.githubusercontent.com/59707859/226135302-954c69e3-65d6-4f9f-8aec-e019b1e14dd5.PNG)

Reply:
Hi @souad988 thanks for your insight. Is really helpful but I have an issue.

![desktop-pry](https://user-images.githubusercontent.com/43345162/226136852-065d36c1-35ea-4f1a-8e24-cc48aaa6b6f8.png)

Ok, this is the way Im seeing this on my desktop. Do you have any suggestion to implement? Because I really don't know how to fix it if I'm not able to see the problem on my screen.

[![Loom video demo](https://cdn.loom.com/sessions/thumbnails/6eb3fca0b03444cf8273eda7fa804bed-with-play.gif)](https://www.loom.com/share/6eb3fca0b03444cf8273eda7fa804bed)

What should I do on a case like this??